lib/scope.js

```diff
@@ -52,5 +62,64 @@ module.exports = {
     // Plain old waiting. Yeah, I abstracted it into here so only
     // one thing was being called at the end
     if ( options.waitForTimeout ) { await scope.page.waitFor( options.waitForTimeout ); }
-  },
+  },  // Ends afterStep
+
+  getSpecifiedChoice: async function getSpecifiedChoice(scope, {ordinal, label_text, group_label_text, roles}) {
+    /* Returns the elementHandle of the field specified by the arguments
+    * 
+    * @param {string} [ordinal='first'] Which instance of the field on the page.
+    * @param {string} [label_text=''] Text of the label for the field, if it has one.
+    * @param {string} [group_label_text=''] Text of the label for the group of fields to which the field belongs, if there is one.
+    * @param {array} role_types 'radio', 'checkbox', or both. The type of field that the developer could be asking for.
```

Comment:
It would be nice to be explicit about which params are optional.

Reply:
I think the square brackets are the syntax for that, but we can also write it in.